content/zh-cn/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -0,0 +1,301 @@
+to escape to the host. Given that the container is running as a non-privileged
+user on the host, it is limited what it can do to the host.
+
+Furthermore, as users on each pod will be mapped to different non-overlapping
+users in the host, it is limited what they can do to other pods too.
+-->
+这种抽象限制了可能发生的情况，例如，如果容器设法逃到宿主机，
+假设容器以宿主机上的一个非特权用户运行，它在宿主机上的操作是受限制的。
+
+此外，由于每个 Pod 上的用户将被映射到宿主机中不同的不重复的用户，他们对其他 Pod 的操作也是受限制的。
+
+<!--
+Capabilities granted to a pod are also limited to the pod user namespace and
+mostly invalid out of it, some are even completely void. Here are two examples:
+- `CAP_SYS_MODULE` does not have any effect if granted to a pod using user
+namespaces, the pod isn't able to load kernel modules.
+- `CAP_SYS_ADMIN` is limited to the pod's user namespace and invalid outside
+of it.
+-->
+赋予 Pod 的功能也仅限于 Pod 用户名字空间，其中大部分无效的，有些甚至完全无效。
+这里有两个例子：
+- `CAP_SYS_MODULE` 如果授予 Pod 使用用户名字空间，则不会起作用，Pod 无法加载内核模块。
+- `CAP_SYS_ADMIN` 仅限于 Pod 的用户名字空间，在外部无效。
+
+<!--
+Without using a user namespace a container running as root, in the case of a
+container breakout, has root privileges on the node. And if some capability were
+granted to the container, the capabilities are valid on the host too. None of
+this is true when we use user namespaces.
+
+If you want to know more details about what changes when user namespaces are in
+use, see `man 7 user_namespaces`.
+-->
+在不使用用户名字空间的情况下，对于以 root 用户运行的容器而言，发生容器逃逸时，
+容器将拥有在宿主机上的 root 特权。如果容器被赋予了某些权限，则这些权限在宿主机上同样有效。
+我们使用用户名字空间后就不会出现这样的问题。
+
+如果你想详细了解关于用户名字空间在使用时会发生什么变化，请参见 `man 7 user_namespaces`。
+
+<!--
+## Set up a node to support user namespaces
+
+It is recommended that the host's files and host's processes use UIDs/GIDs in
+the range of 0-65535.
+ 
+The kubelet will assign UIDs/GIDs higher than that to pods. Therefore, to
+guarantee as much isolation as possible, the UIDs/GIDs used by the host's files
+and host's processes should be in the range 0-65535.
+-->
+## 设置节点支持用户名字空间
+
+建议宿主机的文件和进程使用 UID/GID 的范围为 0-65535。
+
+kubelet 将分配给 Pod 更高的 UID/GID。
+因此，为了尽可能保证隔离，宿主机文件和主机进程使用的 UID/GID 应在 0-65535 范围内。
+
+<!--
+Note that this recommendation is important to mitigate the impact of CVEs like
+[CVE-2021-25741][CVE-2021-25741], where a pod can potentially read arbitrary
+files in the hosts. If the UIDs/GIDs of the pod and the host don't overlap, it
+is limited what a pod would be able to do: the pod UID/GID won't match the
+host's file owner/group.
+-->
+请注意，此建议对于缓解 CVE 的影响非常重要，
+例如 [CVE-2021-25741][CVE-2021-25741]，Pod 可以读取宿主机中的任意文件。
+如果 Pod 和宿主机的 UID/GID 不重叠，
+那么 Pod 能做的事情就会被限制：Pod 的 UID/GID 与宿主机的文件所有者/组不匹配。
+
+[CVE-2021-25741]: https://github.com/kubernetes/kubernetes/issues/104980
+
+<!--
+## Limitations
+
+When using a user namespace for the pod, it is disallowed to use other host
+namespaces. In particular, if you set `hostUsers: false` then you are not
+allowed to set any of:
+
+ * `hostNetwork: true`
+ * `hostIPC: true`
+ * `hostPID: true`
```

Comment:
Line 251 - 253 can be removed when they are not required to localize.

Reply:
OK, I get it, thank you.